paddle/fluid/pybind/op_function.h

```diff
@@ -173,6 +211,708 @@ static inline void HandleViewBetweenInputAndOutput(
             << "), share allocation and inplace version.";
   }
 }
+
+extern PyTypeObject* g_VarBase_PyType;
+extern PyTypeObject* g_VarType_PyType;
+
+inline bool PyObject_CheckBool(PyObject* obj) { return PyBool_Check(obj); }
+
+inline bool PyObject_CheckLong(PyObject* obj) {
+  return (PyLong_Check(obj) && !PyBool_Check(obj)) ||
+         PyObject_IsInstance(obj, (PyObject*)g_VarType_PyType) ||  // NOLINT
+         (_import_array() >= 0 && PyArray_IsScalar((obj), Integer)) ||
+         PyObject_IsInstance(obj, (PyObject*)g_VarBase_PyType);  // NOLINT
+}
+
+inline bool PyObject_CheckFloat(PyObject* obj) {
+  return PyFloat_Check(obj) || PyLong_Check(obj) ||
```

Comment:
Better comment on why PyLong_Check is added.

Reply:
done，thx！